src/driver/safe.rs

```diff
@@ -1121,6 +1121,27 @@ impl CudaDeviceBuilder {
     }
 }
 
+/// Enables profile collection by the active profiling tool for the current context. If profiling is already enabled, then profiler_start() has no effect.
+/// ```
+/// use cudarc::profiler;
```

Comment:
```suggestion
/// ```rust
/// use cudarc::driver::{profiler_start, profiler_stop};
```